app/assets/stylesheets/application.scss

```diff
@@ -20,14 +19,8 @@ $success-color: color("green", "base") !default;
 $error-color: color("red", "base") !default;
 $link-color: color("light-blue", "darken-1") !default;
 
-/*** Navbar ***/
-$navbar-font-color: $brand-font-color;
-
-/*** Button ***/
-$button-color-raised: $primary-font-color !default;
-
 /*** Body ***/
-body { background-color: #E6E0CF; }
+body { background-color: #FAFAFA; }
```

Comment:
Color literals like `#FAFAFA` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FAFAFA` should be written as `#fafafa`